sickbeard/helpers.py

```diff
@@ -1561,3 +1561,18 @@ def canonical_name(obj, fmt='{key}:{value}', separator='|', ignore_list=frozense
     """
     guess = obj if isinstance(obj, dict) else guessit.guessit(obj)
     return str(separator.join([fmt.format(key=k, value=v) for k, v in guess.items() if k not in ignore_list]))
+
+
+def get_broken_providers():
+    """Get broken providers from cdn.pymedusa.com."""
+    url = 'https://cdn.pymedusa.com/providers/broken_providers.txt'
```

Comment:
@p0psicles i stored in the config so if some how user can't connect to cdn, at least he has the "last" broken providers

If you see a better way to do this, feel free to change it.